temoa-uiuc/data_files/02_uiuc.sql

```diff
@@ -1,48 +1,20 @@
 BEGIN TRANSACTION;
-
```

Comment:
I found the comments in this iteration of the .sql file to be very helpful when reading through it.